pkg/rpm/salt-minion

```diff
@@ -25,132 +25,296 @@
 # processname: /usr/bin/salt-minion
 
 
-DEBIAN_VERSION=/etc/debian_version
-SUSE_RELEASE=/etc/SuSE-release
-# Source function library.
-if [ -f $DEBIAN_VERSION ]; then
-   break
-elif [ -f $SUSE_RELEASE -a -r /etc/rc.status ]; then
-    . /etc/rc.status
-else
-    . /etc/rc.d/init.d/functions
-fi
+# Default values (can be overridden in settings file)
+: ${USER:=$(id -nu)}
+SALTMINION="${SALTMINION_BINDIR}/salt-minion"
+SALTCALL="${SALTMINION_BINDIR}/salt-call"
+# SALTMINION_CONFIGS are newline-separated entries of: MINION_USER CONFIG_DIR
+: ${SALTMINION_CONFIGS:="
+$USER ${SALTMINION_SYSCONFDIR}/salt
+"}
+SALTMINION_ARGS=""
+SALTMINION_TIMEOUT=30
+SALTMINION_TICK=1
 
-# Default values (can be overridden below)
-SALTMINION=/usr/bin/salt-minion
-PYTHON=/usr/bin/python
-MINION_ARGS=""
+SERVICE="salt-minion"
+PROCESS="salt-minion"
 
-if [ -f /etc/default/salt ]; then
-    . /etc/default/salt
+# Read in settings file
+if [ -f "${SALTMINION_SYSCONFDIR}/default/salt" ]; then
```

Comment:
`salt-minion` RPM already comes with `/etc/default/salt-minion` file by the way. Why not use it?